boards/nucleo-l1/include/periph_conf.h

```diff
@@ -213,6 +213,46 @@ static const timer_conf_t timer_config[] = {
 #define GPIO_15_IRQ         EXTI15_10_IRQn
 
 /**
+ * @name PWM configuration
+ * @{
+ */
+#define PWM_NUMOF           (2U)
+#define PWM_0_EN            1
+#define PWM_1_EN            1 
+#define PWM_MAX_CHANNELS    4
+
+/* PWM 0 device configuration */
+#define PWM_0_DEV           TIM3
+#define PWM_0_CHANNELS      4
+#define PWM_0_CLK           (168000000U)
+#define PWM_0_CLKEN()       (RCC->APB1ENR |= RCC_APB1ENR_TIM3EN)
+#define PWM_0_CLKDIS()      (RCC->APB1ENR &= ~RCC_APB1ENR_TIM3EN)
+/* PWM 0 pin configuration */
+#define PWM_0_PORT          GPIOB
+#define PWM_0_PORT_CLKEN()  (RCC->AHBENR |= RCC_AHBENR_GPIOBEN)
+#define PWM_0_PIN_CH0       4
```

Comment:
I think if we remove PB4 from the EXTI defines we could remove all the PWM1 defines and have a full functioning PWM_0_DEV :)

@thomaseichinger Is the EXTI line PB4 used for something? Or can it's definition be removed to make place for PWM?